sumpy/e2e.py

```diff
@@ -515,7 +516,9 @@ def get_kernel(self, result_dtype):
     def get_optimized_kernel(self, result_dtype):
         knl = self.get_kernel(result_dtype)
         # FIXME
-        knl = lp.split_iname(knl, "itgt_box", 16, outer_tag="g.0")
+        knl = lp.tag_inames(knl, dict(itgt_box="g.0"))
```

Comment:
How come you decided to leave this outside the M2L transform? Surely it interacts with the code there?

Reply:
Fixed in 2746b9a01e88e749a37308dd0386c6c8ae9ed613